knowledge-base/grid-export-image-column-excel.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Exporting Grid's Image Column to Excel in Blazor
+description: Learn how to export an image column from the Telerik Blazor Grid to an Excel file by utilizing the Document Processing Library.
+type: how-to
+page_title: How to Export Grid's Image Column to Excel with Blazor
+slug: grid-export-image-column-excel
+tags: grid, blazor, telerik, export, excel, image, column
+res_type: kb
+ticketid: 1666986, 1662544, 1645206, 1655440
+---
+
+## Environment
+
+<table>
+<tbody>
+<tr>
+<td>Product</td>
+<td>Grid for Blazor, <br />Document Processing Library</td>
+</tr>
+<tr>
+<td>Version</td>
+<td>6.0.0</td>
+</tr>
+</tbody>
+</table>
+
+## Description
+
+I want to export the Telerik Blazor [Grid]({%slug grid-overview%})'s column that displays images to an Excel file. The images are rendered in a [Grid column template]({%slug grid-templates-column%}). How can I include the image column in the Excel export?
+
```

Comment:
Consider including a "This KB also answers the following questions" section, if applicable.